public/r2/v1/swagger/swagger.json

```diff
@@ -184,6 +184,72 @@
                 }
             }
         },
+        "/namespaces/{namespaceID}/ruleset/bulk": {
+            "put": {
```

Comment:
Should this be POST rather than PUT since calling this endpoint twice with the same set of changes are not idempotent?